diff --git a/src/Overlay.cpp b/src/Overlay.cpp
index c3ae5ae..359cd7e 100644
--- a/src/Overlay.cpp
+++ b/src/Overlay.cpp
@@ -811,7 +811,7 @@ namespace gsr {
 
             window->draw(top_bar_background);
             window->draw(top_bar_text);
-            window->draw(logo_sprite);
+            // window->draw(logo_sprite);
 
             close_button_widget.draw(*window, mgl::vec2f(0.0f, 0.0f));
             page_stack.draw(*window, mgl::vec2f(0.0f, 0.0f));
diff --git a/src/Theme.cpp b/src/Theme.cpp
index 2bef3c8..396e982 100644
--- a/src/Theme.cpp
+++ b/src/Theme.cpp
@@ -29,7 +29,7 @@ namespace gsr {
             vendor = GpuVendor::NVIDIA;
         else if(color_name == "broadcom")
             vendor = GpuVendor::BROADCOM;
-        return gpu_vendor_to_color(vendor);
+        return mgl::Color(89, 193, 191);
     }
 
     bool Theme::set_window_size(mgl::vec2i window_size) {
@@ -188,4 +188,4 @@ namespace gsr {
         assert(color_theme);
         return *color_theme;
     }
-}
\ No newline at end of file
+}
